discretisedfield/tests/test_field.py

```diff
@@ -1731,15 +1731,15 @@ def value_fun(point):
 
     # No PBC
     f = df.Field(mesh_nopbc, nvdim=1, value=value_fun)
-    assert np.allclose(f.diff("x")((11, 1, 1)), 1)
-    assert np.allclose(f.diff("y")((1, 7, 1)), 1)
-    assert np.allclose(f.diff("z")((1, 1, 5)), 1)
+    assert np.allclose(f.diff("x", periodic=False)((11, 1, 1)), 1)
+    assert np.allclose(f.diff("y", periodic=False)((1, 7, 1)), 1)
+    assert np.allclose(f.diff("z", periodic=False)((1, 1, 5)), 1)
```

Comment:
We should probably omit `periodic=False` to test the default and notice accidental changes.